tests/console/system_prepare.pm

```diff
@@ -29,6 +29,7 @@ sub run {
     my ($self) = @_;
     select_console 'root-console';
 
+    disable_serial_getty;
```

Comment:
This addition also breaks test on IPMI HW: https://openqa.suse.de/tests/3402340 (tried multiple time). Fixed since the revert of this PR.

Reply:
I don't see any relation to this though.
It looks like openQA screws up entering the SSH password.
I have no idea how this could have been broken by my changes.